crates/sui-graphql-rpc/src/server/version.rs

```diff
@@ -224,31 +209,28 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn case_insensitive() {
+    async fn named_version() {
```

Comment:
Please also add a test for just plain `/graphql`.

Reply:
Added